### PR TITLE
build input field in components

### LINKED ISSUE #22: build input field in components

### PR DESCRIPTION
**Pull request by mewiemers**
Fixes #22

Deployment: https://meet-albert-mewiemers-i-qql3gp.herokuapp.com/storybook

add input field in storybook
User Story:
user can type in  his name in input field

### CODE REVIEW

File: components/input/Input .tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import styles from "./Input.module.css";
+
+const [name, setName] = useState("");
```
Comment (fre-ben) [inline components/input/Input .tsx]: Not sure, but I think this should be inside the Input() function?

**Review by fre-ben — APPROVED** [review]
LGTM 🧙

File: components/input/Input .tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+export default function Input() {
+  return (
+    <>
```
Comment (RBN2208) [inline components/input/Input .tsx]:
```suggestion


export default function Input() {
const [name, setName] = useState("");
  return (
    <>
```

**Review by RBN2208 — COMMENTED** [review]
Hey, nice! I just left one suggestion, same one as Frederik :)

**Review by RonjaMeinecke — APPROVED** [review]
LGTM 🐨

File: components/input/Input .tsx (hunk 1 above)
Comment (mewiemers) [reply components/input/Input .tsx]: good hint, thanks :-)

**Comment by SilviaBarcellini** [comment]
LGTM 🚀

File: components/input/Input .tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import styles from "./Input.module.css";
+
+export default function Input() {
+  const [name, setName] = useState("");
```
Comment (m-wiemers) [inline components/input/Input .tsx:5]: Storybook is intended for static preview only. Functions and hooks are only used during assembly. I think.

**Review by m-wiemers — APPROVED** [review]
The sun is shining. But even when it laughs, this code is even more beautiful ☀

**Pull request merged**